place_tracker/lib/place_map.dart

```diff
@@ -331,13 +350,13 @@ class PlaceMapState extends State<PlaceMap> {
           children: <Widget>[
             GoogleMap(
               onMapCreated: onMapCreated,
-              options: GoogleMapOptions(
-                trackCameraPosition: true,
-                cameraPosition: CameraPosition(
-                  target: widget.center,
-                  zoom: 11.0,
-                ),
+              initialCameraPosition: CameraPosition(
```

Comment:
This isn't part of the change, but is the Builder above still necessary?

Reply:
Yes, because we still show the snack bar.

Reply:
I'll take your word for it. Normally when I see this pattern, there's a Scaffold constructor called in between the outer build method and the Builder constructor, which makes the need for the Builder obvious.